app/serializers/v1/intervention_serializer.rb

```diff
@@ -11,7 +11,11 @@ class V1::InterventionSerializer < V1Serializer
   has_many :sessions, serializer: V1::SessionSerializer
   has_many :clinic_locations, serializer: V1::ClinicLocationSerializer
 
-  cache_options(store: Rails.cache, namespace: 'intervention-serializer', expires_in: 24.hours)  # temporary length, might be a subject to change
+  cache_options(store: Rails.cache, namespace: 'intervention-serializer', expires_in: 24.hours) # temporary length, might be a subject to change
+
+  attribute :starred do |object, params|
+    object.starred_by?(params[:current_user_id])
```

Comment:
Are you sure that we want to another call to the DB?

Reply:
I don't think it's a huge issue, as it's not a N+1 problem